_docs-v5/view-api/view-object.md

```diff
@@ -34,3 +34,11 @@ A View object contains information about a calendar view, such as title and date
 :   The master Calendar Object this view belongs to.
 
 These properties of the View object can only be read. A [custom view with settings](custom-view-with-settings) is the way to set any of them.
+
+
+## View Types
+
+The available view types currently available are:
```

Comment:
I would rephrase this to: `Here is a list of available view types:`

this list is a bit confusing. I would write it as:

```
- [List Views](link-to-list-views-page)
  - listDay
  - listWeek
  - listMonth
  - listYear
- [TimeGrid Views](link-to-timegrid-view-page)
  - etc
  - etc
```